src/script/page/RightSidebar/RightSidebar.tsx

```diff
@@ -166,6 +166,18 @@ const RightSidebar: FC<RightSidebarProps> = ({
     });
   }, []);
 
+  const containerRef = useCallback(
+    (element: HTMLDivElement | null) => {
+      const focussableElements =
+        'a:not([disabled]), button:not([disabled]), input[type=text]:not([disabled]), [tabindex]:not([disabled]):not([tabindex="-1"])';
```

Comment:
Maybe we should move this string to some const? We're using it in two places now (check out `util.ts` file)

Reply:
+ maybe also create a custom hook for this functionality e.g `useFocusFirstElement` that will return ref callback 🤔, wdyt? seems pretty reusable to me